Packages/io.chainsafe.web3-unity/Samples~/Web3.Unity/Scripts/Scenes/Login.cs

```diff
@@ -58,7 +58,16 @@ protected async Task TryLogin()
 
             LoginSceneIndex = SceneManager.GetActiveScene().buildIndex;
 
-            SceneManager.LoadScene(MainSceneName);
+            // Attempt scene transition based on the login object sceneToLoad value
+            try
+            {
+                SceneManager.LoadScene(sceneToLoad);
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e);
```

Comment:
This should be Debug.LogError, not Console.Writeline

Reply:
done